marklogic-client-api/src/test/java/com/marklogic/client/test/datamovement/JSONSplitterTest.java

```diff
@@ -0,0 +1,241 @@
+package com.marklogic.client.test.datamovement;
```

Comment:
Because the test will be in a public repository, please add the copyright header.

Reply:
Added. Thanks.